mod/scripts/vscripts/takyon_votemap.nut

```diff
@@ -277,11 +279,24 @@ void function SetNextMap(int num, bool force = false){
 int function FindMvdInVoteData(string mapName){ // returns -1 if not found
     int index = -1
     foreach(MapVotesData mvd in voteData){
+        index++
         if(mvd.mapName == mapName)
             return index
-        index++
     }
-    return index
+    return -1
+}
+
+
+int function GetMapVoteCount(string mapName){
```

Comment:
this will never reach the last ```return -1``` wouldn't it be simpler to do something like this:
```
int index = FindMvdInVoteData(mapName)
if(index != -1){
  MapVotesData temp = voteData[index]
  return temp.votes
}
return 0
```

Reply:
I know, but squirrel do not compile without unreachable -1